charts/ccsm-helm/values.yaml

```diff
@@ -23,9 +23,216 @@ global:
   zeebeClusterName: "{{ .Release.Name }}-zeebe"
   zeebePort: 26500
 
+# The values file follows helm best practices https://helm.sh/docs/chart_best_practices/values/
+#
+# This means:
+#   * Variable names should begin with a lowercase letter, and words should be separated with camelcase.
+#   * Every defined property in values.yaml should be documented. The documentation string should begin with the name of the property that it describes, and then give at least a one-sentence description
+#
+# Furthermore, we try to apply the following pattern: # [VarName] [conjunction] [definition]
+#
+# VarName:
+#
+#  * In the documentation the variable name is started with a big letter, similar to kubernetes resource documentation.
+#  * If the variable is part of a subsection/object we use a json path expression (to make it more clear where the variable belongs to).
+#    The root (chart name) is omitted (e.g. zeebe). This is useful for using --set in helm.
+#
+# Conjunction:
+#   * [defines] for mandatory configuration
+#   * [can be used] for optional configuration
+#   * [if true] for toggles
+#   * [configuration] for section/group of variables
+
+# Zeebe configuration for the zeebe sub chart. Contains configuration for the broker and gateway pods and related resources.
```

Comment:
:wrench: The Helm chart best practices still keeps the lowercase first letter of the variable name for the documentation string. I imagine that's to be more consistent, which I think is a good idea. I would also keep it the same (as opposed to this file where it's always uppercase), but don't think it's a big deal.

Reply:
I started with it, but had a look at other charts and the kubernetes docs example https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#scheduling and they always start with a big letter, which is why I added this note https://github.com/camunda-community-hub/camunda-cloud-helm/pull/141/files#diff-65f0fb45844a42a1cdf08ff5bca60e2b36416b7e90d25895f572a04cad76ebcaR36